PROJECT

```diff
@@ -23,14 +23,16 @@ resources:
 - group: cni
   kind: CalicoConfig
   version: v1alpha1
+- group: cni
+  kind: AntreaConfig
+  version: v1alpha1
 - group: run
   kind: TanzuClusterBootstrap
   version: v1alpha3
 - group: run
   kind: TanzuClusterBootstrapTemplate
   version: v1alpha3
-- group: addons
```

Comment:
Looks like this line was added by mistake https://github.com/vmware-tanzu/tanzu-framework/pull/1539/files#diff-d1fb402f1269ba7396dca9cbd03b44ac055874669bce1d0f239fa2f54ea3584bR32